data/morphology/unimorph_languages.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we want the raw github links with the lgs as the only wikipron lg code abbreviation or do we want the format to look like the JSON file format like this example for armenian? 


"hye-am_web_2017_1M": {
        "path": [
            "../tsv/arm_e",
            "../tsv/arm_w"
        ],
        "data_url": "https://raw.githubusercontent.com/reubenraff/wikipron/master/data/tsv/arm_e_phonetic.tsv"
    },

Reply:
We don't need any of that additional metadata here, it seems to me, yet. (UniMorph doesn't distinguish between Western and Eastern Armenian.) We just need the URL of the matching UniMorph file and the language code.

Reply:
ok I'll format as such in the unimorph_data JSON file